pandas/core/generic.py

```diff
@@ -3295,7 +3295,7 @@ def to_latex(
     @doc(storage_options=_shared_docs["storage_options"])
     def to_csv(
         self,
-        path_or_buf: FilePathOrBuffer | None = None,
+        path_or_buf: FilePathOrBuffer[str] | FilePathOrBuffer[bytes] | None = None,
```

Comment:
can't you just use `FilePathOrBuffer[AnyStr]` here?

Reply:
Yes, you are right - thank you!

Reply:
Is it possible to restart `CL/Checks` - it encountered a (transient) conda issue?